pkg/build/node/cni.go

```diff
@@ -27,7 +27,7 @@ const (
 The default CNI manifest and images are our own tiny kindnet
 */
 
-var defaultCNIImages = []string{"kindest/kindnetd:0.5.3@sha256:bc1833b3da442bb639008dd5a62861a0419d3f64b58fce6fb38b749105232555"}
+var defaultCNIImages = []string{"aojea/kindnetd:0.5.4"}
```

Comment:
should a new image be pushed to `kindest/kindnet`?

Reply:
yeah, discussed this PR in the kind meeting today, holding it pending #1124 fix and then pushing a small kind release with assorted small fixes after .6

then this, and also upgrading the debian-ipatables base to the new one coming in k8s 1.17 with much newer iptables